diff --git a/node_modules/@builder.io/qwik-city/lib/vite/index.mjs b/node_modules/@builder.io/qwik-city/lib/vite/index.mjs
index 9d76bdc..d9885fd 100644
--- a/node_modules/@builder.io/qwik-city/lib/vite/index.mjs
+++ b/node_modules/@builder.io/qwik-city/lib/vite/index.mjs
@@ -26206,7 +26206,10 @@ function getQwikCityServerData(requestEv) {
     },
     qwikcity: {
       routeName,
-      ev: requestEv,
+      ev: {
+        url: requestEv.url,
+        originalUrl: requestEv.originalUrl,
+      },
       params: { ...params },
       loadedRoute: getRequestRoute(requestEv),
       response: {
